corehq/apps/app_manager/util.py

```diff
@@ -571,6 +571,11 @@ def get_sort_and_sort_only_columns(detail_columns, sort_elements):
         # Assertion added for enforcing architecture decision & not for ensuring user behavior
         # This should ideally never be false
         assert (sort_element.field or sort_element.sort_calculation)
+
+        # If there is a field present we use that so we can club it with the display property in Suite file.
```

Comment:
Nit: What does "club" mean in this context? I thought it meant "delete", but here it looks like it means "replace" or "pair"?